enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,430 @@
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="3"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="6"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="9"} 0
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="12"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="15"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="18"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="21"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="24"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="27"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="30"} 1
+cluster_installation_bootstrap_bucket{command="create",platform="aws",result="success",le="+Inf"} 1
+cluster_installation_bootstrap_sum{command="create",platform="aws",result="success"} 10
+cluster_installation_bootstrap_count{command="create",platform="aws",result="success"} 1
+```
+
+#### Modification
+The `cluster_installation_modification`- metrics represent whether user-modified manifest or ignition-config assets were used to install the cluster.
+
+The proposed metrics represent categories of one or more asset file modified during an installation:
+
+Metric | Modified Files
+-------|---------------
+cluster_installation_modification_config_manifest | cloud-provider-config.yaml <br> cluster-config.yaml <br> cluster-infrastructure-02-config.yml <br> kube-cloud-config.yaml
+cluster_installation_modification_dns_manifest | cluster-dns-02-config.yml <br> cluster-ingress-02-config.yml
+cluster_installation_modification_network_manifest | cluster-network-01-crd.yml <br> cluster-network-02-config.yml <br> cluster-proxy-01-config.yaml
+cluster_installation_modification_scheduler_manifest | cluster-scheduler-02-config.yml
+cluster_installation_modification_cvo_manifest | cvo-overrides.yaml
+cluster_installation_modification_etcd_manifest | etcd-ca-bundle-configmap.yaml <br> etcd-client-secret.yaml <br> etcd-host-service-endpoints.yaml <br> etcd-host-service.yaml <br> etcd-metric-client-secret.yaml <br> etcd-metric-serving-ca-configmap.yaml <br> etcd-metric-signer-secret.yaml <br> etcd-namespace.yaml <br> etcd-service.yaml <br> etcd-serving-ca-configmap.yaml <br> etcd-signer-secret.yaml
+cluster_installation_modification_machineconfig_manifest | 04-openshift-machine-config-operator.yaml <br> machine-config-server-tls-secret.yaml
+cluster_installation_modification_ca_manifest | machine-config-server-tls-secret.yaml
+cluster_installation_modification_pullsecret_manifest | openshift-config-secret-pull-secret.yaml
+cluster_installation_modification_bootstrap_ignition | bootstrap.ign
```

Comment:
I doubt the installer will reliably have access to the actual bootstrap Ignition config, because it's so easy for a user to pick our generated content up after `create ignition-configs` and then modify it elsewhere on the way to publishing where the bootstrap machine will be able to find it.  Do we want to publish the metric, even though that flow would show up (incorrectly) as a "`bootstrap.ign` was not modified"?  And who would be checking to see if `bootstrap.ign` was modified anyway?  Every `wait-for` command getting invoked?  I'd rather just drop these and have metrics only for files/assets sucked in by the installer in the asset graph leading to the Ignition config files.